NuGet.Config

```diff
@@ -1,6 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
+    <clear />
```

Comment:
Why do we need to avoid machine configured sources? That could break caching improvements on build servers or other local overrides couldn't it?